core/src/main/java/org/elasticsearch/common/util/AbstractHash.java

```diff
@@ -23,7 +23,7 @@
 
 /**
  * Base implementation for {@link BytesRefHash} and {@link LongHash}, or any class that
- * needs to map values to dense ords. This class is not thread-safe.
+ * needs to map values to dense words. This class is not thread-safe.
```

Comment:
I think `ords` rather means something like "ordinals" here. Maybe @jpountz who added the comment can confirm?

Reply:
It means "ordinals" indeed, which we commonly refer to as "ords".